code/modules/economy/ATM.dm

```diff
@@ -34,9 +34,9 @@ log transactions
 	var/view_screen = NO_SCREEN
 	var/datum/effect_system/spark_spread/spark_system
 
-/obj/machinery/atm/New()
-	..()
-	machine_id = "[station_name()] RT #[num_financial_terminals++]"
+/obj/machinery/atm/Initialize()
+	. = ..()
+	machine_id = "[CONFIG_GET(string/ship_name)] RT #[num_financial_terminals++]"
```

Comment:
Since this is also a colony thing it never even made sense to be `station_name()`. Heritage of old Bay/tg.
Either remove, make it dependent on the z level (if AoE it's a NT vending machine, if on the ship a TGMC one), make an ATM company for every case, or something else.